lib/index.js

```diff
@@ -7,17 +7,19 @@ const request = require('request-promise-native')
 const { Cookie } = require('tough-cookie')
 const isUrl = require('is-url')
 const useragentFromSeed = require('useragent-from-seed')
+const getTotp = require('totp-generator');
```

Comment:
It's limiting the usage of the API only with Token. This way if the user has Duo, Google Auth or another authenticator it won't be possible to login with 2FA.

Reply:
The Token you need to give to `totp-generator` is the same token you need to give to Google Auth (although it's called a key there) to generate the 6 digit codes.
`totp-generator` will generate the same 6 digit code that Google Auth would generate.
You can get this token by following [my changes in the readme](https://github.com/jlobos/instagram-web-api/pull/169/commits/01cb66d5b2e75848cdf22de778079fa89ebd1de0#diff-04c6e90faac2675aa89e2176d2eec7d8).

I've decided to use this token, because it is permanent (compared to the six digit code, which is valid for 30 seconds only), so it can be used in scripts.

Maybe TOTP Token isn't the best name for it (I don't know if it's called that elsewhere). Would `twofa_secret` be a better name for it? I am open to suggestions :)